ui/src/app/components/gke-app-configuration-panel.tsx

```diff
@@ -65,7 +67,7 @@ export const GKEAppConfigurationPanel = ({
     useState<Error>();
 
   const [panelContent, setPanelContent] = useState<GKEAppPanelContent>(
-    GKEAppPanelContent.CREATE
+    initialPanelContent || GKEAppPanelContent.CREATE
```

Comment:
nit/TOL: why || and not ??

Reply:
Because I forgot `??` exists! Converted 👍